src/components/Book.js

```diff
@@ -1,14 +1,22 @@
 import React from 'react';
-import PropTypes from 'prop-types';
+// import PropTypes from 'prop-types';
+import { useDispatch, useSelector } from 'react-redux';
+import { removeBook } from '../redux/books/Books';
 
-const Book = (props) => {
-  const { book } = props;
+const Book = () => {
+  // const { book } = props;
+  const book = useSelector((state) => state.books);
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧